assignments/pr_review_tutorial/chernyavskaya/README.md

```diff
@@ -1,13 +1,11 @@
-Ok so I guess you are reading this cuz you wanna use my code. There are some
-functions that do stuf and thats:
+There are some two functions in simple_functions.py, "factorial" and "fibonacci".
+Factorial gives the factorial (the "!" function) of the argument:
 
     >>> from simple_functions import factorial
     >>> factorial(10)
-    9
+    3,628,800
```

Comment:
If code testing is enabled here, all of these code snippets would run in a python interpreter, and would return an error if the outputs don't match. I suggest to use the output format of the interpreter, e.g. have an integer without the formatting commas.